.goreleaser.yml

```diff
@@ -49,3 +50,83 @@ brews:
       fish_completion.install "completions/cloud-platform.fish"
     dependencies:
       - name: go
+dockers:
+  - image_templates: [ "ministryofjustice/{{ .ProjectName }}:{{ .Tag}}-amd64" ]
+    goarch: amd64
+    dockerfile: '{{ .Env.DOCKERFILE }}'
+    use: buildx
+    build_flag_templates:
+      - --build-arg=COMMAND={{ .ProjectName }}
+      - --platform=linux/amd64
+      - --label=org.opencontainers.image.title={{ .ProjectName }}
+      - --label=org.opencontainers.image.description={{ .ProjectName }}
+      - --label=org.opencontainers.image.url=https://github.com/ministryofjustice/{{ .ProjectName }}
+      - --label=org.opencontainers.image.source=https://github.com/ministryofjustice/{{ .ProjectName }}
+      - --label=org.opencontainers.image.version={{ .Version }}
+      - --label=org.opencontainers.image.created={{ time "2006-01-02T15:04:05Z07:00" }}
+      - --label=org.opencontainers.image.revision={{ .FullCommit }}
+      - --label=org.opencontainers.image.licenses=MIT License
```

Comment:
We must build with this env var set containing the latest image tag. Otherwise, it'll screw with the cli's version precheck.

CLOUD_PLATFORM_CLI_VERSION=${{ steps.metadata.outputs.tags }}

Reply:
edited the build_flag_templates to include CLOUD_PLATFORM_CLI_VERISON={{ .Tag }} tag is equal to the current release tag

--build-arg=CLOUD_PLATFORM_CLI_VERSION={{ .Tag}}